src/Routes/DetailPage/DetailPage.tsx

```diff
@@ -55,8 +56,15 @@ const DetailPage = () => {
 
   console.log('INFO:DetailPage render:domain_id=' + domain_id);
 
+  if (appContext.rbac.isLoading) {
```

Comment:
I think this code block is the root cause for the "Rendered more hooks than during the previous render".  The root cause seems to be that we are breaking the "Rules of hooks". https://react.dev/warnings/invalid-hook-call-warning 

Mainly the part "use Hooks at the top level of your React function, **before any early returns**"

Moving the code-block with the "return" after all the `useXXX` makes it more stable.

The same applies for all the other pages where this happened.